src/main/resources/db/migration/V2_0__RECREATE_TIER_CALCULATION.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS TIER_CALCULATION;
```

Comment:
obviously here we'll lose data,
We can clear all the databases before we go live and consolidate these scripts.

Reply:
If we get this in prod before testing starts it will be fine I think.